code/modules/cm_marines/vehicle_part_fabricator.dm

```diff
@@ -134,86 +144,100 @@
 /obj/structure/machinery/part_fabricator/dropship/ui_static_data(mob/user)
 	var/list/static_data = list()
 	static_data["Equipment"] = list()
+	var/is_ammo = 0
+	var/index = 1
 	for(var/build_type in typesof(/obj/structure/dropship_equipment))
-		var/obj/structure/dropship_equipment/DE = build_type
-		if(SSticker.mode && MODE_HAS_TOGGLEABLE_FLAG(MODE_NO_COMBAT_CAS) && initial(DE.combat_equipment))
+		var/obj/structure/dropship_equipment/dropship_equipment_data = build_type
+		if(SSticker.mode && MODE_HAS_TOGGLEABLE_FLAG(MODE_NO_COMBAT_CAS) && dropship_equipment_data.combat_equipment)
+			index +=  1
 			continue
-		var/build_name = initial(DE.name)
-		var/build_description = initial(DE.desc)
-		var/build_cost = initial(DE.point_cost)
+		var/build_name = dropship_equipment_data.name
+		var/build_description = dropship_equipment_data.desc
+		var/build_cost = dropship_equipment_data.point_cost
```

Comment:
I'm sorry if this was confusing in the past, but if you're still going to read the variables from type paths, then yes you're going to have to use initial. Be sure to run your branch locally and you'll see it throws this runtime when trying to set up the static data:

![image](https://github.com/cmss13-devs/cmss13/assets/76988376/d3bbc996-6ebb-43de-aee3-299936f7375b)